QSB/Animation/AnimationSync.cs

```diff
@@ -112,55 +144,30 @@ private void InitCrouchSync()
             _crouchSync.Init(this, _playerController, _bodyAnim);
         }
 
-        private void OnJump() => SendTrigger(AnimTrigger.Jump);
-        private void OnBecomeGrounded() => SendTrigger(AnimTrigger.Grounded);
-        private void OnBecomeUngrounded() => SendTrigger(AnimTrigger.Ungrounded);
-
-        private void OnSuitUp() => SendTrigger(AnimTrigger.SuitUp);
-        private void OnSuitDown() => SendTrigger(AnimTrigger.SuitDown);
+        private void OnJump() => _netAnim.SetTrigger("Jump");
+        private void OnBecomeGrounded() => _netAnim.SetTrigger("Grounded");
+        private void OnBecomeUngrounded() => _netAnim.SetTrigger("Ungrounded");
 
-        public void SendTrigger(AnimTrigger trigger, float value = 0)
+        public void SendCrouch(float value = 0)
```

Comment:
Crouching should ofc be extracted to new class, but you don't *have* to do that in this PR ;)